runtime/runtime-params-estimator/src/estimator_context.rs

```diff
@@ -383,10 +427,8 @@ impl Testbed<'_> {
 
     /// Instantiate a new trie for the estimator.
     fn trie(&mut self) -> near_store::Trie {
-        self.tries.get_trie_with_block_hash_for_shard(
-            ShardUId::single_shard(),
-            self.root,
-            &FLAT_STATE_HEAD,
-        )
+        let num_deltas = self.config.finality_lag;
+        let tip = CryptoHash::hash_borsh(num_deltas);
```

Comment:
also a nit, so feel free to keep it as it is : the same here, I would be nice to explicitly pass `tip` here as a last generated block in `generate_deltas`